.github/workflows/release.yml

```diff
@@ -51,13 +51,21 @@ jobs:
           ./gradlew release -Prelease.dryRun
           echo "::set-output name=tagName::$(./gradlew cV -q -Prelease.quiet)"
 
-      - name: Create Github release
-        uses: "marvinpinto/action-automatic-releases@latest"
+      - name: Extract release notes
+        id: extract-release-notes
+        uses: ffurrer2/extract-release-notes@v1
```

Comment:
@bastiandoetsch, did you check whether it's possible to use `/gradlew getChangelog --console=plain -q --no-header` to extract changelog without additional GH action?

Reply:
done